### PR TITLE
feat: power dashboard changes

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

File: cmd/collectors/zapi/plugins/sensor/sensor.go [hunk 1]
```diff
@@ -180,9 +180,9 @@ func (my *Sensor) calculateEnvironmentMetrics(data *matrix.Matrix) ([]*matrix.Ma
 				if len(v.powerSensor) > 0 {
 					for _, v1 := range v.powerSensor {
 						if v1.unit == "mW" {
-							sumPower += sumPower + (v1.value / 1000)
+							sumPower += v1.value / 1000
```
Comment (rahulguptajss) [inline cmd/collectors/zapi/plugins/sensor/sensor.go:183]: That was it!! :-(

**Review by Hardikl — APPROVED** [review]

File: cmd/collectors/zapi/plugins/sensor/sensor.go (hunk 1 above)
Comment (cgrinds) [reply cmd/collectors/zapi/plugins/sensor/sensor.go:183]: classic

File: grafana/dashboards/cmode/harvest_dashboard_power.json [hunk 2]
```diff
@@ -519,8 +555,12 @@
       "id": 82,
       "options": {
         "legend": {
-          "calcs": [],
-          "displayMode": "list",
+          "calcs": [
+            "max",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/harvest_dashboard_power.json:559]: order should be mean, last, max

File: grafana/dashboards/cmode/harvest_dashboard_power.json [hunk 3]
```diff
@@ -622,8 +662,12 @@
       "id": 73,
       "options": {
         "legend": {
-          "calcs": [],
-          "displayMode": "list",
+          "calcs": [
+            "max",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/harvest_dashboard_power.json:666]: order should be mean, last, max

File: grafana/dashboards/cmode/harvest_dashboard_power.json [hunk 4]
```diff
@@ -882,8 +934,12 @@
       "id": 80,
       "options": {
         "legend": {
-          "calcs": [],
-          "displayMode": "list",
+          "calcs": [
+            "max",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/harvest_dashboard_power.json:938]: order should be mean, last, max

File: grafana/dashboards/cmode/harvest_dashboard_power.json [hunk 5]
```diff
@@ -1802,7 +1858,7 @@
       "pluginVersion": "8.1.2",
       "targets": [
         {
-          "expr": "shelf_labels{datacenter=\"$Datacenter\",cluster=\"$Cluster\"}",
+          "expr": "shelf_labels{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\"}",
```
[the remark this answers is not in the document]
Comment (rahulguptajss) [reply grafana/dashboards/cmode/harvest_dashboard_power.json:1861]: I'll change it. Must be from on fly changes

**Comment by rahulguptajss** [comment]
addressed review comments

**Review by cgrinds — APPROVED** [review]
👍

**Pull request merged**